csun-metalab/nomi
# HOTFIX-11-20-18-fixingCors

# Description
Fix upload functionality and CORS issue.
  
# Testing
1. Visit the student list and student profile modals and ensure that upload functionality works.
2. Ensure there is no console errors after playing around and messing with the upload functionality.
3. Also test it on the test environment.
